src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -3215,91 +3237,9 @@ import ageDisplay, { getLowerAgeYears } from '@ncigdc/utils/ageDisplay';
 
   InCHlib.prototype._download_img = function (img_format) {
```

Comment:
Refactoring this function to be async, so we can allow animations to happen meanwhile.
The user should not be allowed to add any more tasks to avoid crashing the app, as generating images has proven rather "expensive".